[ADD] WhatsApp: Documentation

tiku-odoo opened the pull request:
Add WhatsApp configuration documentation

Dev PR https://github.com/odoo/enterprise/pull/45601#pullrequestreview-1608996613
Task [3495962](https://www.odoo.com/web#cids=3&model=project.task&view_type=form&id=3495962)

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/5762).

tiku-odoo commented:
Ready for review: 
@StraubCreative @jcs-odoo

tiku-odoo commented:
CC: @larm-odoo @Felicious 

Creds for DB (don't make changes please :)

DB setup on: https://tiku-sali-odoo-test.odoo.com/

try it out!

UN: testkukulka@yahoo.com
PW: Summer2022!

StraubCreative commented:
@tiku-odoo can you address the build error(s) please: https://runbot.odoo.com/runbot/build/50906185

StraubCreative commented:
@larm-odoo @jero-odoo can you review this right away please. I want to be able to look at this and r+ immediately, thank you.

tiku-odoo commented:
Moved file location to productivity per @jcs-odoo and added section on Whatsapp template errors. 

Functionality documentation still needs to be ironed out. 

@StraubCreative the build errors are out of my control I believe-- github error and errors in other docs. Can you have a look?

larm-odoo commented:
Hi @tiku-odoo - my review was still open while you did another commit, so my comments above say they are 'outdated' but they can still be implemented =)  Sam had to help me when I got the error just now. I didn't get through all of it, so you'll only see the comments for the parts I got through- doing the rest now and I'll ping you when I am done. I will try to be as fast as possible, but I am trying to be thorough- and those two things don't go together =D

jcs-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+messages can be received into the database from the customer. There is no setup required to use this
+version of WhatsApp with Odoo.
+
+The second version is that of a Meta business account, where the company's Meta account is
+configured in Odoo via a :abbr:`API (Application Programming Interface)` connection. This
+document will cover the second scenario.
```
I'd focus on this default configuration, and  add the "Odoo Online" info as a short note afterwards.

larm-odoo reviewed (changes requested):
Here's the second batch before you update branches/push a commit!

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+
+Next the administrator will configure the WhatsApp API by adding a phone number and sending a first
+test message. If the browser isn't on the :guilabel:`Quickstart` page for WhatsApp navigate to
+`<https://developers.facebook.com/apps>`_ and click on the app that is being configured. In the left
```
```suggestion
`<https://developers.facebook.com/apps>` and click on the app that is being configured (the app name is Odoo if the instructions above were followed) . ```

jcs-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+WhatsApp in Odoo
+================
+
+WhatsApp is a messaging service operated by Meta. It is commonly used as a communication tool in
+many countries and within many businesses. This documentation will cover the integration of a
+WhatsApp Business Account with Odoo.
```
Can you reuse the intro on the current page (and improve it if you'd like)?
https://github.com/odoo/documentation/blob/saas-16.3/content/applications/productivity/whatsapp.rst?plain=1

IMO, it better distinguishes WhatsApp, Odoo WhatsApp and WhatsApp Business Account. The list also explains straight away what's doable with this app.

tiku-odoo replied:
I've moved the page over! Thanks for the suggestion :)

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+Next the administrator will configure the WhatsApp API by adding a phone number and sending a first
+test message. If the browser isn't on the :guilabel:`Quickstart` page for WhatsApp navigate to
+`<https://developers.facebook.com/apps>`_ and click on the app that is being configured. In the left
+menu under :guilabel:`WhatsApp` expand the menu by clicking on the `V` icon.
```
```suggestion
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+`<https://developers.facebook.com/apps>`_ and click on the app that is being configured. In the left
+menu under :guilabel:`WhatsApp` expand the menu by clicking on the `V` icon.
+
+A small menu will reveal itself with the following options: :guilabel:`Quickstart`, :guilabel:`API
```
```suggestion
On the left hand side, click the `V` icon next to :guilabel:`WhatsApp` to expand the menu. A small menu will reveal itself with the following options: :guilabel:`Quickstart`, :guilabel:`API
```

jcs-odoo reviewed (commented):
Hi @tiku-odoo :)

- [Already done] Your branch should target saas-16.3, as WhatsApp won't be backported to 16.0, to my knowledge.
- [Already done] In terms of structure, please move your content to the existing [application/productivity/whatsapp.rst doc](https://github.com/odoo/documentation/blob/saas-16.3/content/applications/productivity/whatsapp.rst?plain=1). There is already an intro that explains what we can and can not do with it, but that's pretty much it (although there are some useful external links as well)
  - I see why you put it in "maintain" the same way we'd configure a mail server, but since it won't be only about how to set up a configuration, but also how to use it (concept of templates, the link with PoS, events and such) I think it's best to keep it in the user doc, as it is the case now
- In my opinion, we don't have to explain at length the process on the Meta's side. That would reduce the length and complexity of our documentation, and reduce the maintenance cost of this documentation (we never know when there will be a change on Meta's side).
  - for example this 
     > To create a Business account with Meta/Facebook navigate to: https://business.facebook.com/overview. Click Create account and then enter a name for the business, the administrator’s name, and a work email address. Finally click Next to complete the setup process. Congratulations the Meta business account has been created
     
      could be "if you do not have a Meta Business account, create one on https://business.facebook.com/overview" + seealso link. But it's up to you, just sharing my opinion on this =)

Cheers, and thanks for your work

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+menu under :guilabel:`WhatsApp` expand the menu by clicking on the `V` icon.
+
+A small menu will reveal itself with the following options: :guilabel:`Quickstart`, :guilabel:`API
+Setup`, and :guilabel:`Configuration`. Click on :guilabel:`Quickstart` and then click on
```
```suggestion
Setup`, and :guilabel:`Configuration`. Click :guilabel:`Quickstart` and then click
```

larm-odoo replied:
This is where I am stuck by following along. I can't advertise or whatever the error was when I tried to link the business account I made. So from here on out, I can try and follow along but I will mostly have to review for... grammar/clarity/etc because I can't finish setting this up on my own in my test environment.

larm-odoo reviewed (changes requested):
Another set- I got another outdated message, so I have to submit this now.

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+:guilabel:`Authentication`. In most instances the first two options will be used, unless the user
+would like to send a password reset or something security related.
+
+Enter the :guilabel:`Name` of the template and select the :guilabel:`Language` for the template.
```
```suggestion
Enter a :guilabel:`Name` for the template, then select the :guilabel:`Language'.
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+Enter the :guilabel:`Name` of the template and select the :guilabel:`Language` for the template.
+
+.. note::
+   Multiple languages can be selected by typing and selecting another as needed.
```
```suggestion
   Multiple languages can be selected by typing the language name(s) and selecting the other languages as needed.
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+         highlighted.
+
+After making the appropriate selections click on :guilabel:`Continue` in the upper right hand
+corner. The page will redirect to the :guilabel:`Edit template` page. Here the :guilabel:`Header`,
```
```suggestion
corner. The page redirects to the :guilabel:`Edit template` page. Here the :guilabel:`Header`,
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+After making the appropriate selections click on :guilabel:`Continue` in the upper right hand
+corner. The page will redirect to the :guilabel:`Edit template` page. Here the :guilabel:`Header`,
+:guilabel:`Body`, :guilabel:`Footer` and :guilabel:`Buttons` can be configured. To the right of the
```
```suggestion
:guilabel:`Body`, :guilabel:`Footer` and :guilabel:`Buttons` are configured. To the right of the
```

larm-odoo reviewed (changes requested):
Another set for you!

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+First navigate to  `<https://business.facebook.com/wa/manage/home>`_. Then go to
+:menuselection:`Account tools --> Message templates`. Click on one and then :guilabel:`Edit
+template` or create a new template.
```
```suggestion
template`.  If a new template is needed, create a new template by clicking :guilabel: `Create`.
```

I can't follow along here, so I am not sure if this is correct- but if the default/usual way is to edit a template maybe stick to just that scenario, or add a note hat this can be done on a new template?

tiku-odoo replied:
Should be 'new'

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+.. seealso::
+   :ref:`whatsapp/templates`
+
+After the template has been submitted and approved by Meta, the template needs to be synced with
```
```suggestion
After the template has been submitted and approved by Meta, sync the template with
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+   :ref:`whatsapp/templates`
+
+After the template has been submitted and approved by Meta, the template needs to be synced with
+Odoo. To sync the template see this documentation: :ref:`whatsapp/sync`.
```
If this is referencing the sync section above, I'd have an anchor to the sync section, then have the words 'sync the template with Odoo' as the link to that section instead of having a doc link here.

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+Once the template(s) has been uploaded to the Odoo database navigate to :menuselection:`WhatsApp app
+--> Templates` and open the template that should be modified. Click :guilabel:`Reset to draft` to
+edit the template. The dynamic variable(s) need to be configured on the Odoo side as well to specify
```
```suggestion
edit the template. The dynamic variable(s) are configured on the Odoo side as well to specify
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+To edit the dynamic variables on a template, first change the :guilabel:`Type` to :guilabel:`Field
+of Model`. This allows Odoo to reference a field within a model to produce unique data in the
+end message.
```
```suggestion
message being sent.
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+field are referenced.
+
+.. tip::
+   By default Meta uses the :guilabel:`Applies to` field set to the :guilabel:`Contact` model.
```
```suggestion
   Meta has the :guilabel:`Applies to` field set to the :guilabel:`Contact` model by default.
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+.. tip::
+   By default Meta uses the :guilabel:`Applies to` field set to the :guilabel:`Contact` model.
+
+To search the available fields, simply type in the front-end name (on user-interface) of the field
```
```suggestion
To search the available fields, type in the front-end name (on user-interface) of the field
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+   By default Meta uses the :guilabel:`Applies to` field set to the :guilabel:`Contact` model.
+
+To search the available fields, simply type in the front-end name (on user-interface) of the field
+in the search. This will find a result from all of the available fields for the model
```
```suggestion
in the search box. This will find a result from all of the available fields for the model
```


If it's not a box, change to whatever it is (field, etc)

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+.. warning::
+   Customizing WhatsApp templates is out of the scope of Odoo Support.
+
+After updating the dynamic variables in the template it needs to be submitted to Meta for approval
```
```suggestion
After updating the dynamic variables on the template, the template needs to be submitted to Meta for approval
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+   Customizing WhatsApp templates is out of the scope of Odoo Support.
+
+After updating the dynamic variables in the template it needs to be submitted to Meta for approval
+once again. Click :guilabel:`Submit for Approval` to start the approval process. An email will be
```
```suggestion
again. Click :guilabel:`Submit for Approval` to start the approval process. An email will be
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+once again. Click :guilabel:`Submit for Approval` to start the approval process. An email will be
+sent to the administrator of the Meta account when the template has been approved.
+
+Following the approval from Meta, the templates need to be synced once again on the Odoo database.
```
```suggestion
Following the approval from Meta, sync the templates again in the Odoo database.
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+=============
+
+Notifications in WhatsApp are handled similar to a message conversation in Odoo. A pop-up window
+will appear with the received conversation from the customer. By default notifications are set in
```
```suggestion
appears with the received conversation from the customer. By default, notifications are set in
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+WhatsApp channel.
+
+.. tip::
+   Set up multiple WhatsApp accounts for different departments. For example, the helpdesk team and
```
```suggestion
   It is recommended to set up multiple WhatsApp accounts for different departments. For example, the helpdesk team and
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+.. tip::
+   Set up multiple WhatsApp accounts for different departments. For example, the helpdesk team and
+   sales team can chat on different channels.
```
```suggestion
   sales team can chat on different channels. There is no limit to the amount of accounts that can be created.
```

Check if that is true! But we should specify if it is unlimited or caps at a certain number like 10, 25, etc.

tiku-odoo replied:
Omitting this edit.

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+.. note::
+   Once a conversation is initiated between a user and a customer, notifications to all the users
+   specified in the WhatsApp business account configuration won't take place. Only notifications to
+   the user in the conversation will occur. Should the user not respond within 15 days the
```
```suggestion
   the user in the conversation will occur. Should the user not respond within 15 days, the
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+.. note::
+   Once a conversation is initiated between a user and a customer, notifications to all the users
+   specified in the WhatsApp business account configuration won't take place. Only notifications to
```
```suggestion
   specified in the WhatsApp business account configuration cannot be modified for the current conversation. Only notifications to
```

I was confused by this and unsure what you meant- I believe once the conversation starts, the notifications can't be modified/ Is that true? If t, just clarify what can and can't be modified and when.

tiku-odoo replied:
the notifications won't occur to other users

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+   customer's reply will populate once again to all the users specified in the WhatsApp
+   configuration.
+
+Adding users to chat
```
```suggestion
Adding users to a chat
```

tiku-odoo replied:
omitting per Zac's previous comments

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+Adding users to chat
+====================
+
+Users can be added to the WhatsApp chat by expanding the WhatsApp pop-up window. WhatsApp
```
```suggestion
Users can be added to a WhatsApp chat by expanding the WhatsApp pop-up window. WhatsApp
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+Users can be added to the WhatsApp chat by expanding the WhatsApp pop-up window. WhatsApp
+conversations are located in the *Discuss* app. Click on the person icon with the plus [ + ] icon
+next to it. A window will appear to invite users to the conversation.
```
```suggestion
next to it. A window appears to invite users to the conversation.
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+Validation error
+~~~~~~~~~~~~~~~~
+
+If a pre-configured template from demo-data is submitted for approval to Meta a validation error
```
```suggestion
If a pre-configured template from demo-data is submitted for approval to Meta, a validation error
```

larm-odoo reviewed (changes requested):
This was such a thorough and amazing doc- i really liked the FAQ section with common issues and things to know! If you have any questions n what I wrote or why, just let me know. 

Great work, Tim!!!!

larm-odoo commented:
Hi @tiku-odoo - I finished my review. I had to do it in many chunks, and I was prompted to submit the review even after I thought it was submitted, so if things are doubled up for some reason, I apologize. This was the first time I had to do a review while edits were being done/commits done at the same time, so this was different =)  Most of my changes were just to change to present tense, and/or to clarify some things to be even more clear.  It looks like a lot but most of the edits were like that. I could not follow along entirely since I didn't want to mess up the DB as requested, but I was able to folow along in the FB side for a bit with my fake business =)

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+many countries and within many businesses. This documentation will cover the integration of a
+WhatsApp Business Account with Odoo.
+
+There are two different setups that WhatsApp can have in Odoo. The first is for Odoo Online users
```
Took this section out as Out-of-box Whatsapp doesn't work currently.

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+
+There are two different setups that WhatsApp can have in Odoo. The first is for Odoo Online users
+(SaaS); this version works out-of-box and there are a set number of messages that can be sent and no
+messages can be received into the database from the customer. There is no setup required to use this
```
Removed section

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -0,0 +1,569 @@
+-------------------------------
+
+Now that the basic structure of the app has been created a product will need to be added to the app.
+To access the Meta App Dashboard go to `<https://developers.facebook.com/apps>`_ and click on the
```
This underscore needs to be here for an external link.

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+receive messages` and click the dropdown next to :guilabel:`To`, under :guilabel:`Step 1 Select
+phone numbers`.
+
+Now, select the only option available: :guilabel:`Manage phone number list`. Adding a phone number
```
Not making this change.

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+
+The next step that must be configured is within Odoo. A few different values for a token, phone
+number and account IDs need to be configured in Odoo in order to created a :guilabel:`Callback URL`
+and :guilabel:`Webhook Verify Token` to configure the Webhooks and in order to receive messages back
```
Confusing, changed the wording

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+WhatsApp --> API Configuration`. Copy the following values from the Meta developer console into
+Odoo:
+
+- :guilabel:`Phone number ID` --> :guilabel:`Phone Number ID`
```
I've added in a table to clarify this

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+- :guilabel:`WhatsApp Business Account ID` --> :guilabel:`Account ID`
+
+To retreive the :guilabel:`App Secret`, navigate to the Meta developer console,
+`<https://developers.facebook.com/apps>`_ and select the app that Odoo is being configured in. Then
```
This is a required character in Odoo docs @larm-odoo 

See this doc: 
https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#hyperlinksv

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,583 @@ content. Businesses can use `WhatsApp Business
+   <https://developers.facebook.com/docs/whatsapp/cloud-api/get-started/add-a-phone-number>`_.
+
+..
+  This section will be needed at a later date when it is working in Odoo.
```
The section was grey because I commented it out, meaning it doesn't appear in the render, only the code. @larm-odoo

tiku-odoo commented:
@jero-odoo This doc is ready for your immediate review. I've changed the templates section after @larm-odoo reviewed it due to a fixed traceback and the templates working. Feel free to let me know if you have any questions. I appreciate the thorough review.

tiku-odoo commented:
@LNAodoo This is the PR for WhatsApp documentation-- can you review it for accuracy? Specifically the templates section. Thank you!

tiku-odoo commented:
@larm-odoo Can you review the templates section once more when you have a moment? I've made significant changes to the flow-- streamlining it.

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+
+Assets must be added to the system user and then a permanent token can be generated. Click on
+:guilabel:`Add assets`. When the pop-up window appears select :guilabel:`Apps` under the
+:guilabel:`Select asset type`. Then select the Odoo app. Toggle the permissions to on under
```
```suggestion
:guilabel:`Select asset type`. Then select the Odoo app. Toggle the permissions to *On* under
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+:guilabel:`Development` to :guilabel:`Live`.
+
+.. note::
+   If the app status  is not set to *live*, then the database will only be able to contact the test
```
```suggestion
   If the app status is not set to *live*, then the database will only be able to contact the test
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+overview the process for creating templates in Odoo.
+
+.. important::
+   WhatsApp has an approval process that must be completed before the template can be used.
```
May be worth linking to the doc explaining the approval process (or to the section below where you explain it) in case a reader sees this warning but has additional questions.

tiku-odoo replied:
Added anchor

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+Creating templates
+------------------
+
+Templates are prefabricated layouts that allow users to professional looking messages to customers.
```
```suggestion
Templates are prefabricated layouts that allow users to send professional looking messages to customers.
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+   :guilabel:`Model` name in the backend. Search (using front-end name) for this model in the
+   :guilabel:`Applies to` drop-down menu in the WhatsApp template.
+
+Often when changing the model or :guilabel:`Applies to` field the :guilabel:`Phone Field` made
```
```suggestion
Often when changing the model or :guilabel:`Applies to` field the :guilabel:`Phone Field` may
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+the template is created for. There may be multiple levels that need to be configured.
+
+Change the :guilabel:`Category` to fit either a :guilabel:`Marketing`, :guilabel:`Utility`, or
+:guilabel:`Authentication` category. Specifying an incorrect category can cause a flag from Meta In
```
```suggestion
:guilabel:`Authentication` category. Specifying an incorrect category can cause a flag from Meta in
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+:guilabel:`Authentication` category. Specifying an incorrect category can cause a flag from Meta In
+the approval process.
+
+Add and :guilabel:`User` that are allowed to use this template. On the right hand column of the top
```
```suggestion
Add any :guilabel:`Users` that are allowed to use this template. On the right hand column of the top
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+:guilabel:`Type` to :guilabel:`Field of Model`. This allows Odoo to reference a field within a model
+to produce unique data in the message being sent.
+
+Next, go to the :guilabel:`Variables` tab and edit the :guilabel:`Field` field of the dynamic
```
```suggestion
Next, edit the :guilabel:`Field` field of the dynamic
```
They should already be on that tab

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+to produce unique data in the message being sent.
+
+Next, go to the :guilabel:`Variables` tab and edit the :guilabel:`Field` field of the dynamic
+variables. The :guilabel:`Applies to` field in the template can be edited prior to ensure the
```
```suggestion
variables. The :guilabel:`Applies to` field in the template should be edited prior to ensure the
```

jero-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+WhatsApp channel.
+
+.. tip::
+   It is recommended to set up multiple WhatsApp accounts for different departments. For example,
```
I would move this tip above the configuration section.

jero-odoo reviewed (changes requested):
nice work @tiku-odoo. Just a few suggestions. One thing I did want to bring up is since this will be on the documentation for 16, it is probably worth adding a note that this integration is only available for 16.3 and 16.4 databases.

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+
+The second version is that of a Meta business account, where the company's Meta account is
+configured in Odoo via a :abbr:`API (Application Programming Interface)` connection. This
+document will cover the second scenario.
```
This has been modified as the first scenario isn't done yet.

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+templates (validated on the WhatsApp side) and through open form communication (by the user).
+
+First an app will need to be created on Meta's developer console. Then the Odoo database will need
+to be connected to the app.
```
noted thanks.

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+Setup of Meta business account
+------------------------------
+
+To create a Business account with Meta/Facebook navigate to:
```
noted, I've added this in

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+`<https://business.facebook.com/overview>`_. Click :guilabel:`Create account` and then enter a name
+for the business, your name, and a work email address. Finally click :guilabel:`Next` to complete
+the setup process. Congratulations the Meta business account has been created.
+
```
noted, I've streamlined this block

tiku-odoo commented on content/administration/maintain/whatsapp.rst:
```diff
@@ -0,0 +1,527 @@
+   If the business account is linked to a personal Facebook account then the administrator must
+   toggle between the personal account to the business account for the remainder of the
+   configuration. To toggle to the business account navigate to `<https://developers.facebook.com>`_
+   and click on the *account name* in the upper right corner. Under :guilabel:`Business Accounts`
```
account name is not what is seen on the GUI

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+   - `Meta Business: connect a phone number to the WhatsApp Business account <https://www.facebook.com/business/help/456220311516626>`_
+   - `Meta Business: change the WhatsApp Business display name <https://www.facebook.com/business/help/378834799515077>`_
+
+WhatsApp is a messaging service operated by Meta. It is commonly used as a communication tool in
```
Noted thanks!

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+approval` button in the upper left hand corner. The status of the template will change to
+:guilabel:`Pending`. The status will remain in :guilabel:`Pending` until a decision has been made by
+Meta. Once the user receives an email confirming that the template has been approved the templates
+will need to be synced from the Odoo database.
```
```suggestion
must be synced from the Odoo database.
```

i feel "must" is strong, and it won't work unless they do it, so it seemed fitting.

tiku-odoo replied:
It is a must do

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+See this documentation: :ref:`whatsapp/sync`.
+
+.. tip::
+   Pre-configured demo-data templates can be submitted to Meta for approval. Navigate to
```
```suggestion
   There are pre-configured demo-data templates available in Odoo to use or modify. These templates can be used 
   as-is or modified to suit a specific business's needs. To use these templates, Navigate to
```

tiku-odoo replied:
Edited via a different comment.

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+
+.. tip::
+   Pre-configured demo-data templates can be submitted to Meta for approval. Navigate to
+   :menuselection:`WhatsApp app --> Templates` and select the pre-configured template. Click
```
```suggestion
   :menuselection:`WhatsApp app --> Templates` and select a pre-configured template. Click
```

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+--------------------------------
+
+Dynamic variables reference certain fields within the Odoo database to produce unique data in the
+WhatsApp message when using a template. Dynamic variables are encoded to display fields from within
```
```suggestion
configured WhatsApp message templates. Dynamic variables are encoded to display fields from within
```

tiku-odoo replied:
Not going to implement it, unclear

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` field on the :guilabel:`Sales Order` model.
+
+Dynamic variables can be added in to the :guilabel:`Body` by adding :guilabel:`placeholders` in the
```
Maybe this can be anchored to where I commented above about a sample message?

tiku-odoo replied:
Not going to anchor a sentence only headings

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+   model by settings a dynamic variable. For example, a customer's name can be referenced in the
+   email from the :guilabel:`Customer` field on the :guilabel:`Sales Order` model.
+
+Dynamic variables can be added in to the :guilabel:`Body` by adding :guilabel:`placeholders` in the
```
```suggestion
At the bottom of the template  form, there are three tabs, :guilabel:`Body`, :guilabel:`Buttons`, and :guilabel:`Variables`. These three tabs combined create the template. The text is entered into the :guilabel:`Body` tab, and dynamic content that is called out in the :guilabel:`Body` tab is specified in the :guilabel:`Variables` tab. Every piece of dynamic content (variables) in the message (body) is specifically called out and specified in the :guilabel:`Variables` tab.

For example, in the body tab, if the following is typed in the body tab: "Hello {{1}}," , then "{{1}}" must be described in the variables tab, and the information that needs to be pulled to populate the field labeled {{1}} must be configured in the variables tab. For this specific case, the message should greet the customer by name, so the {{1}} shoul dbe configured to populate the {{1}} filed with the :guilabel:`User Name`.

Dynamic variables can be added in to the :guilabel:`Body` by adding :guilabel:`placeholders` in the
```


I am not 100% sure this works with the following 2 sentences, but I think it is very important to make it clear that the text is calling out items form the other tab (it took e a minute to figure this out). Plus, it would benefit form an image with the tabs shown. Or maybe with both tabs side by side, and an arrow pointing out the configured info for one of the placeholders. It's not a very complicated concept but for someone first seeing it, like me, it could be very confusing at first. This might help.

tiku-odoo replied:
Resolved, added modification of this in.

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+correct model and field are referenced.
+
+To search the available fields, type in the front-end name (on user-interface) of the field in the
+search box. This will find a result from all of the available fields for the model
```
Again, I don't see where/how to search? I might be missing something but this could be specified/cleared up, because I was lost.

tiku-odoo replied:
Just need to select the box and type in it. A search box appears

larm-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+approval again. Click :guilabel:`Submit for Approval` to start the approval process. An email will
+be sent to the administrator of the Meta account when the template has been approved.
+
+Following the approval from Meta, sync the templates again in the Odoo database. See this
```
Yu probably don't know, but are the pre-configured templates "pre-approved" from Meta in any way? I just wondered how long this process typically is, and if it is faster if using the pre-configured ones. Not sure if support would know this or not. Not necessary info but if it's available it would be good to add. (Like does Meta have an approval pending dashboard or something we can reference? Just so we can point user's to see the status and have an idea about the process))

tiku-odoo replied:
Added link to Meta portal

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+Under the :guilabel:`Account` select the *WhatsApp business account* in Odoo that this template
+should link to. Next, under the :guilabel:`Applies to` field select the *model* the server action
+for this template, will *apply to*.
+
```
It's an access rights issue for that user.

tiku-odoo replied:
Noted, I've added a link to access rights and a disclaimer

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+
+Often when changing the model or :guilabel:`Applies to` field the :guilabel:`Phone Field` made
+produce an error. The :guilabel:`Phone field `should always result in `Phone` or `Mobile`. To search
+the available fields, type in the front-end name (on user-interface) of the field in the search box.
```
Added in image this should be in the same paragraph

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+produce an error. The :guilabel:`Phone field `should always result in `Phone` or `Mobile`. To search
+the available fields, type in the front-end name (on user-interface) of the field in the search box.
+This will find a result from all of the available fields for the model (:guilabel:`Applies to`) that
+the template is created for. There may be multiple levels that need to be configured.
```
I've clarified.

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+- Image
+- Video
+- Document
+- Location
```
Variables need to be set for the location.

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+- Document
+- Location
+
+Navigate to the :guilabel:`Body` tab to configure the main message of the template.
```
There is an example below already!

tiku-odoo commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,665 @@ content. Businesses can use `WhatsApp Business
+
+Navigate to the :guilabel:`Body` tab to configure the main message of the template.
+
+When all the necessary changes are made to the template, then click on the :guilabel:`Submit for
```
Already noted. Not going to get into these for this doc.

tiku-odoo commented:
@larm-odoo Can you approve this PR? 

@StraubCreative This PR is ready for your review. 

Thanks :+1:

larm-odoo reviewed (approved):
Sorry Tim- I don't remember approving docs, just reviewing =)

StraubCreative commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,786 @@ content. Businesses can use `WhatsApp Business
+Meta account is configured in Odoo via a :abbr:`API (Application Programming Interface)` connection.
+
+The Whatsapp connector supports two flows: company initiated, and customer initiated. A company can
+initiate a discussion by sending a template to one or a batch of people. Once the template is sent,
```
grammar
(it's not "one... people", it's one _person_)
1.
```suggestion
initiate a discussion by sending a template to one or a batch of persons. Once the template is sent,
```
2.
```suggestion
initiate a discussion by sending a template to one or more people. Once the template is sent, the
recipient can answer to trigger a discussion between the sender and the receiver (a discuss chat
```

StraubCreative commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -7,17 +7,786 @@ content. Businesses can use `WhatsApp Business
+
+The Whatsapp connector supports two flows: company initiated, and customer initiated. A company can
+initiate a discussion by sending a template to one or a batch of people. Once the template is sent,
+the recipient can answer to trigger a discussion between the sender and the receiver (a discuss chat
+window will popup if the customer answers within 15 days).
```
wording
- cause>effect
- popup is a noun; pop up is the action you're looking for 😉 

*Discuss* is capitalized and italicized here since it's an Odoo product, no?

```suggestion
recipient can answer in order to trigger a discussion between the sender and the receiver (a
*Discuss* chat window will pop up if the customer answers within 15 days).
```

StraubCreative reviewed (changes requested):
Hi @tiku-odoo @jero-odoo @larm-odoo 

Great work with the content y'all. I'm going to stop making comments on my review in order to save time, and instead just address items I'm seeing, which mostly revolve around wording and grammar. Will push up those fixes in a separate commit soon! 💪

StraubCreative commented:
Overview of changes
================
Applied final content review batch changes on: [c18d052](https://github.com/odoo/documentation/pull/5762/commits/c18d05267d6c7e0cb9aca409c79143a06e6dbfe7).

Improved grammar, wording, specificity, tags and formatting, etc.
Full commit diff detailing changes here: https://github.com/odoo/documentation/compare/2e8fb36..c18d052

Follow-up notes
============
For your consideration @tiku-odoo @jero-odoo @larm-odoo : 

REVIEW: Line 322: the **Webhook fields** section *appears* to be incorrect, or at least needs further clarification I think.

The information we have listed on this doc doesn't seem to match both this [overview doc on webhooks](https://developers.facebook.com/docs/graph-api/webhooks/getting-started#configure-webhooks-product) or this [WhatsApp specific one](https://developers.facebook.com/docs/graph-api/webhooks/getting-started/webhooks-for-whatsapp), both of which, for example, tell users to **Subscribe to app objects** in order to create and manage specific webhook fields (which we don't mention).

I played around on the Meta dev console > app UI as well and do not see some field labels or buttons as indicated in our article. For example, there doesn't appear to be a **Webhook fields** section title anywhere on the Meta dev console UI?

I left the section mostly as it is, however, given how sensitive Meta's tools are especially in regard to configuration, I'd recommend we double-check the accuracy and clarity here.

-----

OPTIONAL/LATER:  **Line 385: payment warning.** This warning felt like it came out of nowhere and isn't specific as to *why* a payment method is needed, and for *what*.

-----

OPTIONAL/LATER:  **Line 480: Creating templates in Odoo section.** There *seems* to be a lot of unnecessary interchanging between the terms **model** and **field**. Probably could use another once through just to ensure that we're being specific and accurate when discussing these technical concepts.

-----

OPTIONAL/LATER:  **Line 554: define the list items.** These should get guis too since they're selectable menu items.

-----

Line 674: {{6}} variable is missing a **Field** value. Was this intentional?

Next steps
========
- Perform any/all necessary changes labeled REVIEW listed above @tiku-odoo @larm-odoo @jero-odoo 
- @samueljlieber perform a final technical review after to ensure all links are working, tags are correct, etc.
- We can include FP's addition from #5853 as well in the publication
- Tag me to merge right after everything above is complete (pending any further comments from @jcs-odoo @LNAodoo et al. 😉 )

tiku-odoo commented:
> REVIEW: Line 322: the **Webhook fields** section _appears_ to be incorrect, or at least needs further clarification I think.
> 
> The information we have listed on this doc doesn't seem to match both this [overview doc on webhooks](https://developers.facebook.com/docs/graph-api/webhooks/getting-started#configure-webhooks-product) or this [WhatsApp specific one](https://developers.facebook.com/docs/graph-api/webhooks/getting-started/webhooks-for-whatsapp), both of which, for example, tell users to **Subscribe to app objects** in order to create and manage specific webhook fields (which we don't mention).
> 
> I played around on the Meta dev console > app UI as well and do not see some field labels or buttons as indicated in our article. For example, there doesn't appear to be a **Webhook fields** section title anywhere on the Meta dev console UI?
>

**Webhooks will only appear for configuration once the app is created. Then once the subscription is confirmed using the Callback URL and Webhook Verify Token then the webhooks fields can be set. I've added in a screenshot for both webhooks sections and also a note stating this.** 

> 
> I left the section mostly as it is, however, given how sensitive Meta's tools are especially in regard to configuration, I'd recommend we double-check the accuracy and clarity here.
> 
> OPTIONAL/LATER: **Line 385: payment warning.** This warning felt like it came out of nowhere and isn't specific as to _why_ a payment method is needed, and for _what_.
>

**This is part of Meta fraud detection system, they want to make sure your company is real. I was flagged immediately here and that was the only way to move on. I've added in a note.**

> 
> OPTIONAL/LATER: **Line 480: Creating templates in Odoo section.** There _seems_ to be a lot of unnecessary interchanging between the terms **model** and **field**. Probably could use another once through just to ensure that we're being specific and accurate when discussing these technical concepts.
> 

**I re-read and don't see any errors.**

> OPTIONAL/LATER: **Line 554: define the list items.** These should get guis too since they're selectable menu items.
> 
> REVIEW: Line 674: {{6}} variable is missing a **Field** value. Was this intentional?
> # Next steps
> 

**Yes this is how the template is configured, it is a Portal Link not a field.** 

>     * Perform any/all necessary changes labeled REVIEW listed above @tiku-odoo @larm-odoo @jero-odoo
> 
>     * Tag @samueljlieber to perform a final technical review after to ensure all links are working, tags are correct, etc.
> 
>     * We can include FP's addition from [Update whatsapp.rst #5853](https://github.com/odoo/documentation/pull/5853) as well in the publication
> 
>     * Tag me to merge right after everything above is complete (pending any further comments from @jcs-odoo @LNAodoo et al. 😉 )

@samueljlieber This doc is ready for your review. Thanks.

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+      :alt: Toggle between Meta personal and business accounts.
+
+.. important::
+  In order to create a Meta business account, the user must already have a personal Facebook account
+  that has existed for a minimum of one hour prior to setting up the Facebook Business account.
+  Trying to create the business account prior to this time will result in an error.
```
Needs another space for indent here
```suggestion
   In order to create a Meta business account, the user must already have a personal Facebook
   account that has existed for a minimum of one hour prior to setting up the Facebook Business
   account. Trying to create the business account prior to this time will result in an error.
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+are then used to configure the webhooks (in order to receive messages back into the database).
+
+In Odoo, navigate to :menuselection:`WhatsApp app --> Configuration --> WhatsApp Business Accounts`.
+THen click :guilabel:`New` to configure the WhatsApp business account in Odoo.
```
Correction
```suggestion
Then click :guilabel:`New` to configure the WhatsApp business account in Odoo.
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+Messages`.
+
+Copy and paste the :guilabel:`Callback URL` from Odoo into the :guilabel:`Callback URL` field in
+Meta. Similiarly, copy and paste the :guilabel:`Webhook Verify Token` into the :guilabel:`Verify
```
Correction
```suggestion
Meta. Similarly, copy and paste the :guilabel:`Webhook Verify Token` into the :guilabel:`Verify
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+the screen that follows.
+
+With that token value, update the :guilabel:`Access Token` field in the WhatsApp business account in
+Odoo by navigating to :menuselection:` WhatsApp app --> Configuration --> WhatsApp Business
+Accounts `
```
Syntax errors
```suggestion
Odoo by navigating to :menuselection:`WhatsApp app --> Configuration --> WhatsApp Business
Accounts`.
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+.. warning::
+   A privacy policy URL must be set in order for the app to be set to live. Go to the Meta developer
+   console, `<https://developers.facebook.com/apps>`_ and select the app that Odoo is being
+   configured in. Then, using the menu on the left side of the screen, go to :menuselection:` App
```
Syntax errors
```suggestion
   configured in. Then, using the menu on the left side of the screen, go to :menuselection:`App
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+
+.. important::
+   WhatsApp has an approval process that must be completed before the template can be used.
+   :ref:`productivity/whatsapp/approval`
```
Missing period 
```suggestion
   :ref:`productivity/whatsapp/approval`.
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+
+   Dear {{1}},
+
+   Here is your invoice *{{2}}* from *{{3}}* for a total of *{{4}}{{5}}*.
+   To review your invoice or pay online: {{6}}
```
Force line breaks to display like it is in the template
```suggestion
   | Here is your invoice *{{2}}* from *{{3}}* for a total of *{{4}}{{5}}*.
   | To review your invoice or pay online: {{6}}
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+        - Field of Model
+        - `Amount`
+      * - body - {{6}}
+        - https://..
```
Escape the link so it formats as plain text
```suggestion
        - \https://..
```

samueljlieber commented on content/applications/productivity/whatsapp.rst:
```diff
@@ -2,22 +2,886 @@
+<https://business.facebook.com/wa/manage/home>`_, and then go to :menuselection:`Account tools -->
+Message templates`.
+
+   .. image:: whatsapp/account-tools.png
+      :align: center
+      :alt: Account tools highlighted in business manager with the manage templates link
+            highlighted.
```
No need for this image to be indented
```suggestion
.. image:: whatsapp/account-tools.png
   :align: center
   :alt: Account tools highlighted in business manager with the manage templates link highlighted.
```

samueljlieber commented on content/applications/productivity/whatsapp/add-variable.png:
This image is unused in the document

samueljlieber commented on content/applications/productivity/whatsapp/variables.png:
This image is unused in the document

samueljlieber reviewed (approved):
Hi @tiku-odoo, great job with getting this doc together, and props to everyone involved 💪. 

FYI in https://github.com/odoo/documentation/commit/026f08714f66487d337faff05f78de4090f395bb I had to revert to @StraubCreative's [commit](https://github.com/odoo/documentation/pull/5762#issuecomment-1728834562) because his changes were not included in [your commit](https://github.com/odoo/documentation/pull/5762#issuecomment-1729697346), I applied your changes in [my commit](https://github.com/odoo/documentation/commit/026f08714f66487d337faff05f78de4090f395bb) as well. All work is restored. 

Below are my technical corrections, which I will push up in another commit after this review. Please see them for reference and I will approve once my commit is made and checks pass.

Thank you!

samueljlieber commented:
Corrected commit message so all co-authors are included in https://github.com/odoo/documentation/commit/60fddedec722a4c81ec1c15dbd68f616d0ca65d5

StraubCreative commented:
Final checks, rebase, amend commit message on [1adb5d7](https://github.com/odoo/documentation/pull/5762/commits/1adb5d7991895b633dbab3ee151a82d4d4a30c78).

All checks pass ✅ 
Good job everyone!

@robodoo r+
